projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/select-docs/select-docs.component.html

```diff
@@ -298,6 +298,34 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
         </div>
       </div>
 
+      <div class="go-column go-column--100 go-column--no-padding">
+        <div class="go-container">
+          <div class="go-column go-column--100">
+            If need be, you can also remove the "Select All" button using the
+            <code class="code-block--inline">@Input() showSelecAll: boolean = false;</code> binding.
+          </div>
+          <div class="go-column go-column--50">
+            <h4 class="go-heading-4 go-heading--underlined">View</h4>
+            <go-select
+              bindLabel="name"
+              bindValue="value"
+              [control]="select7"
```

Comment:
Should this have its own control? If multiple selects have the same control it will change the value for both of them if either of them are changed right?

Reply:
Yeah, I put up a new PR to fix these #509